config/configgrpc/configgrpc.go

```diff
@@ -76,11 +76,11 @@ type GRPCClientSettings struct {
 	// (https://godoc.org/google.golang.org/grpc#WithKeepaliveParams).
 	Keepalive *KeepaliveClientConfig `mapstructure:"keepalive"`
 
-	// The WriteBufferSize for client gRPC. See grpc.WithReadBufferSize
+	// ReadBufferSize for gRPC server. See grpc.WithReadBufferSize
```

Comment:
Should be `client` instead as it's `GRPCClientSettings` struct

Reply:
Done.